scripts/audit-ci-tests.js

```
@@ -0,0 +1,301 @@
+        const content = await fs.readFile(filePath, 'utf-8')
+        const ciRelated = this.isCIRelated(content, filePath)
+        const ciAspects = ciRelated ? this.extractCIAspects(content) : undefined
+        testFiles.push({
+          path: filePath,
+          content,
+          ciRelated,
+          ciAspects,
+        })
+      }
+    } catch (error) {
+      console.error('Error discovering test files:', error)
```

ESLint

[error] 'console' is not defined.

(no-undef)